src/plugins/discover/public/application/components/table/table.tsx

```diff
@@ -101,9 +101,8 @@ export function DocViewTable({
               ? 'nested'
               : indexPattern.fields.getByName(field)?.type;
             return (
-              <React.Fragment>
+              <React.Fragment key={field}>
                 <DocViewTableRow
-                  key={field}
```

Comment:
I'm surprised this didn't work. How come?

Reply:
Also didn't know: 
https://reactjs.org/docs/fragments.html#keyed-fragments